WordPressKit/WordPressComOAuthClient.swift

```diff
@@ -124,6 +124,7 @@ public final class WordPressComOAuthClient: NSObject {
     ///     - multifactorCode: Multifactor Authentication One-Time-Password. If not needed, can be nil
     ///     - success: block to be called if authentication was successful. The OAuth2 token is passed as a parameter.
     ///     - failure: block to be called if authentication failed. The error object is passed as a parameter.
+    @available(*, deprecated, message: "Here for legacy compatiblity. Use the verions with explicit `needsMultifactor` parameter.")
```

Comment:
Thank you for the suggestion @Ecarrion 

![image](https://github.com/wordpress-mobile/WordPressKit-iOS/assets/1218433/646f0af4-645a-4a5a-bc63-d62d1194f74a)